### PR TITLE
MINOR: Adding changes to update jekyll 

### PR DESCRIPTION
**Pull request by bbejeck**
### Description

Changes for updating jekyll version from `3.9` to `4.2.0`

### Staging Docs

 http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/MINOR_update_jekyll_version/

### CODE REVIEW

**Comment by bbejeck** [comment]
@ybyzek  and @colinhicks can you take a quick look?

File: _includes/tutorials/finding-distinct/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+kstreams/code/tutorial-steps/dev/outputs/
```
Comment (ybyzek) [inline _includes/tutorials/finding-distinct/.gitignore:1]: @bbejeck is this line intended to be part of this PR?

Comment (bbejeck) [inline _includes/tutorials/finding-distinct/.gitignore:1]: adding this as a quick piggy-back fix

**Review by ybyzek — APPROVED** [review]
@bbejeck approved, assuming the site builds and tests pass. Please note, I left one comment to ask if one line is necessary, otherwise LGTM

File: _includes/tutorials/finding-distinct/.gitignore (hunk 1 above)
Comment (bbejeck) [reply _includes/tutorials/finding-distinct/.gitignore:1]: Yes, see my comment below.  I found this by running a test for an issue the `kstreams/code/tutorial-steps/dev/outputs/` directory contains the output generated by the harness-runner and shouldn't get checked-in. cf https://github.com/confluentinc/kafka-tutorials/issues/729

**Comment by bbejeck** [comment]
The site builds and deploys successfully.  The test failures are unrelated as the only changes made are for using a newer version of `jekyll` and associated dependencies for rendering the site.

**Comment by bbejeck** [comment]
Site deployed and all tests passed!

**Review by rspurgeon — APPROVED** [review]
LGTM, verified with `bundle update jekyll` and `bundle exec jekyll servce --livereload` without error

**Comment by bbejeck** [comment]
Thanks for the reviews @ybyzek and @rspurgeon !

**Pull request merged**